api/bucketsd/client/client.go

```diff
@@ -73,7 +72,7 @@ func (c *Client) Root(ctx context.Context, key string) (*pb.RootResponse, error)
 func (c *Client) Links(ctx context.Context, key, pth string) (*pb.LinksResponse, error) {
 	return c.c.Links(ctx, &pb.LinksRequest{
 		Key:  key,
-		Path: normalizePath(pth),
+		Path: filepath.ToSlash(pth),
```

Comment:
Switching these back since on windows this `ToSlash` method functions just like the custom func I needed to test the behavior on macOS.